### PR TITLE
zr-asmft-393-Spanish-Toggle

### PR DESCRIPTION
**Pull request by roppel5ls**
This is to test the toggle feature, switching between our Spanish and English sites on legalshield.com

This test also found a bug with MicrosoftEdge that is currently being investigated by the Spanish team. Edge will fail, as the toggle does not show up.

### CODE REVIEW

File: demo/SPANISH_regressionSuite/spanish_corp.js [hunk 1]
```diff
@@ -53,4 +57,31 @@ what('Test "Get A Plan" & "Get Started Today" to confirm Checkout Loads',functio
...
+  browser.setViewportSize({
+         width: 1400,
+         height: 1050
+     })
+  browser.url('https://uat.legalshield.com/')
+  browser.pause(2500)
+  click('#lang-section-desktop > span:nth-child(3) > span:nth-child(1) > span:nth-child(3) > i:nth-child(1)')
+  browser.keys("ArrowDown")
+  browser.keys("Enter")
+  browser.pause(2500)
```
Comment (TrentonWalkup) [inline demo/SPANISH_regressionSuite/spanish_corp.js]: pause and waitUntil?

**Comment by Gregorma** [comment]
LGTM

**Comment by Gregorma** [comment]
LGTM again

**Comment by roppel5ls** [comment]
Approved - @TrentonWalkup @Gregorma

**Pull request merged**